loading_test.go

```diff
@@ -37,7 +37,7 @@ func TestLoadFromHTTP(t *testing.T) {
 
 	ts2 := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
 		rw.WriteHeader(http.StatusOK)
-		rw.Write([]byte("the content"))
+		_, _ = rw.Write([]byte("the content"))
```

Comment:
An interesting case of this most pedantic linting engine not being consistent with itself (e.g. CLI linter vs hosted by CI)

Reply:
hi!
It's because golangci.com uses the latest version of golangci-lint in which we've disabled default excludes for gosec (gas). I've reverted this change in the latest golangci-lint and we will update golangci.com to use it.

Reply:
@jirfag Thank you for your interest.
In the end, what will be the recommended practice for: 
`rw.Write()`

The previous version told us to put everywhere this: `_,_ = rw.Write()` or `_ = json.Unmarshal()`
And now it is the opposite...

So in the end, what is the code expected by golangci?

Reply:
explicit ignoring of errors by `_ = errFunc()` is ok now. We have 2 linters checking errors: `errcheck` and `gosec`. `errcheck` by default allows such ignoring of errors, but `gosec` doesn't. Therefore for a long time we've manually excluded such issues from `gosec` issues. Few days ago we've decided to disable `gosec` by default and removed such excludes for errors checking. But we didn't think thank a lot of users use `enable-all` in `.golangci.yml`, like you. So we've reverted this change and `gosec` doesn't report anything about errors now.